src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -312,6 +312,12 @@ private function readPropertiesUntil(array $zval, PropertyPathInterface $propert
             $property = $propertyPath->getElement($i);
             $isIndex = $propertyPath->isIndex($i);
 
+            $isNullSafe = false;
+            if (method_exists($propertyPath, 'isNullSafe')) {
+                // To be removed in symfony 6 once we are sure isNullSafe is always implemented.
+                $isNullSafe = $propertyPath->isNullSafe($i);
+            }
```

Comment:
I think we need to add an `else` clause here to raise a deprecation notice to help people with upgrading their code.